app/assets/stylesheets/modules/scan-or-deliver.scss

```diff
@@ -41,3 +41,40 @@
     }
   }
 }
+
+.panel-danger {
+  .panel-heading {
+    border-bottom: 0;
+    font-size: 1.3em;
+
+    a {
+      border-bottom: 0;
+    }
+
+    .glyphicon {
+      line-height: 1.3em;
+    }
+
+    .glyphicon-chevron-down {
```

Comment:
I'm not thrilled by this, but couldn't find it in me to find the bootstrap-collapse-twiddling code we've done elsewhere.. at least until we decide whether we **can** collapse it, or if we're showing this note at all.